RELEASE_NOTES.md

```diff
@@ -1,15 +1,43 @@
 # Release Notes
 
-## testing-1.20-4.0.0-rc.4, v4.0.0-rc.4
+## stable-1.20-4.0.0, v4.0.0
+
+### ambassador
+  - bump ambasador to 1.12
```

Comment:
```suggestion
  - bump ambassador to 1.12
```

Reply:
It's also a completely valueless note and should be removed.